src/components/m-dropdown/mixins/dropdown.scss

```diff
@@ -148,6 +148,9 @@
 @mixin dropdown-toggle() {
   position: relative;
 
+  // this is important to enable click events on iOS
+  cursor: pointer;
```

Comment:
mmmmm maybe better separate this inanother mixing (single responsability)

Reply:
well, every toggle button needs it anyway. Cause even at desktop the expected style is `pointer` on hover.

Reply:
or do mean to extract those 2 lines into a mixin and include it here?